packages/pages-shared/package.json

```diff
@@ -22,11 +22,13 @@
 	},
 	"devDependencies": {
 		"@cloudflare/workers-tsconfig": "workspace:*",
+		"@miniflare/storage-memory": "^2.14.2",
 		"@cloudflare/workers-types": "^4.20240419.0",
-		"@types/service-worker-mock": "^2.0.1",
+		"@miniflare/cache": "^2.14.2",
+		"@miniflare/core": "^2.14.2",
+		"@miniflare/html-rewriter": "^2.14.2",
```

Comment:
These changes feel wrong to me. This means that this whole package starts depending upon the old deprecated Miniflare packages, right?

Reply:
OK, so happy to depend upon html-rewriter since @mrbbot approved. But why all these other `@miniflare` packages?